frontend/src/lib/utils.tsx

```diff
@@ -737,7 +738,25 @@ export const formatDateRange = (dateFrom: dayjs.Dayjs, dateTo: dayjs.Dayjs, form
     return `${dateFrom.format(formatFrom)} - ${dateTo.format(formatTo)}`
 }
 
-export const dateMapping: DateMappingOption[] = [
+// Hourly date mappings for real time analytics. Only shown on Trends at the moment.
+// Could potentially be expensive to compute.
+export const realTimeDateMapping: DateMappingOption[] = [
+    {
+        key: 'Last Hour',
```

Comment:
We use "Sentence case" pretty much everywhere in the product:
```suggestion
        key: 'Last hour',
```